static/app/types/profiling/core.tsx

```diff
@@ -51,18 +52,16 @@ type Percentiles = {
 
 export type FunctionCall = {
   duration_ns: Percentiles;
-  duration_values: number[];
   frequency: Percentiles;
-  frequency_values: number[];
   image: string;
   key: string;
   line: number;
-  main_thread_percent: Record<string, number>;
+  main_thread_percent: number;
   path: string;
   profile_id_to_thread_id: Record<string, number>;
   profile_ids: string[];
   symbol: string;
-  transaction_names;
```

Comment:
whoa, what :D? Is this just `any` if you type it like that :D?

Reply:
idk lol, i spotted this as i was cleaning things up and was shook this ever worked